opensfm/commands/extract_metadata.py

```diff
@@ -25,20 +27,29 @@ def run(self, args):
         if data.exif_overrides_exists():
             exif_overrides = data.load_exif_overrides()
 
-        camera_models = {}
-        for image in data.images():
-            if data.exif_exists(image):
-                logging.info('Loading existing EXIF for {}'.format(image))
-                d = data.load_exif(image)
-            else:
-                logging.info('Extracting EXIF for {}'.format(image))
+        def extract_exif(image):
+            if not data.exif_exists(image):
+                log.setup()
+                logger.info('Extracting EXIF for {}'.format(image))
                 d = self._extract_exif(image, data)
 
                 if image in exif_overrides:
                     d.update(exif_overrides[image])
 
                 data.save_exif(image, d)
 
+        processes = min(data.config['processes'], 2) # Don't use all CPUs as we're I/O bound
```

Comment:
I'm guessing it is i/o bound only if it needs to decode the entire image

Reply:
I think you are right.